make/run-cicd-script.mk

```diff
@@ -1,4 +1,4 @@
-OWNER_AND_BRANCH_LOCATION=codeready-toolchain/toolchain-cicd/master
+OWNER_AND_BRANCH_LOCATION=mfrancisc/toolchain-cicd/build_platform_config
```

Comment:
reminder

Reply:
yes, I'll update this as soon as https://github.com/codeready-toolchain/toolchain-cicd/pull/74 is merged.

Thanks!